src/initramfs/files/init.sh

```
@@ -23,41 +27,127 @@ get_option() {
 
 # device can be specified by partition and fs labels, but we will use only partition label
 get_device() {
-    local LABEL_NAME="${1#*=}"; # 'LABEL=rootfs' > 'rootfs
+    LABEL_NAME="${1#*=}"; # 'LABEL=rootfs' > 'rootfs
     blkid -t PARTLABEL="$LABEL_NAME" --output device || echo;
 }
 
+_log() {
+    echo -e "$1: $0: $2" > /dev/console 2>&1
```

Shellcheck (0.11.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)